holmes/plugins/prompts/_general_instructions.jinja2

```diff
@@ -1,5 +1,8 @@
 # In general
 
+{% if cluster_name -%}
```

Comment:
Thinking here loud, we think it will have also impact when work on issues which aren't k8s? 

If not, do you think that moving it to the `If investigating Kubernetes problems` will be more relevant?